.github/ISSUE_TEMPLATE/NEW_SYNTAX.md

```diff
@@ -20,7 +20,7 @@ assignees: ''
 
 **Please provide the TC39 URL for the syntax proposal:**
 
-
+<!-- for example, <https://github.com/tc39/proposal-decorators> -->
```

Comment:
good to replace it with a finished proposal - in general, we do not accept stage<4 features. it's a bit confusing as decorator is still in stage-2.

Reply:
Valid point. I changed it to the stage 4 `top-level-await` proposal.